roles/confluent.common/tasks/main.yml

```diff
@@ -50,6 +50,16 @@
     mode: 0755
   when: jolokia_enabled|bool
 
+- name: Copy Jolokia Jar
+  copy: 
+    src: "{{ jolokia_jar_url }}"
```

Comment:
So the jolokia_jar_url variable in this case can be a path? We should at least update the annotation in `roles/confluent.common/defaults/main.yml` to mention that.

Also run this
```
python doc.py
```
To update the variables.md file. same goes for the jmxexporter_jar_url annotation